test/rules/typedef/all/test.ts.lint

```diff
@@ -40,3 +40,25 @@ var NoTypesFn = function (
          ~         [expected variable-declaration: 'c' to have a typedef]
         d = a - b;
          ~         [expected variable-declaration: 'd' to have a typedef]
+
+class TsLint {
+    helloWorld() {
+               ~   [expected call-signature: 'helloWorld' to have a typedef]
+        return 'Hello World';
+    };
+
+    arrowHelloWorld = () => {
+                   ~          [expected member-variable-declaration: 'arrowHelloWorld' to have a typedef]
+                       ~      [expected arrow-call-signature to have a typedef]
+        return this.helloWorld();
+    };
+}
+
+setTimeout(() => {}, 1000);
+setTimeout(function() {}, 1000);
```

Comment:
`arrow-parameter` option don't warn this line. Is this `arrow-parameter` option bug?